micro_epsilon_scancontrol_msgs/srv/GetTime.srv

```diff
@@ -0,0 +1,3 @@
+---
+uint32 time
+int32 return_code
```

Comment:
Please add some comments to explain what everything is and means, same for the other services

Reply:
Oh and no newline is present :)

Reply:
Added comments and a newline for the new messages. The old messages still do not have a newline. I will probably create a separate PR for that to avoid confusion